bin/oay/Cargo.toml

```diff
@@ -51,7 +51,7 @@ dav-server-opendalfs = { path = "../../integrations/dav-server", optional = true
 dirs = "5.0.1"
 futures = "0.3"
 futures-util = { version = "0.3.29", optional = true }
-opendal.workspace = true
+opendal = { path="../../core" }
```

Comment:
IIRC when publish crate to cargo, it requires all deps are not `path` or under the same workspace. Do we break the rules here or we don't publish this crate and thus should add a `publish = false` property?

Reply:
Yep. My current plan is to replace `{ path="../../core" }` with the correct version during publish.

Reply:
Sounds good. For releasing different artifacts separately, we may need a script to run commands for all packages and for dedicated packages ..